deploy/helm/sumologic/upgrade-1.0.0.sh

```diff
@@ -194,6 +195,75 @@ then
   yq w -i new.yaml fluentd.logs.containers.extraFilterPluginConf "$FILTER"
 fi
 
+# Prometheus changes
+# Diff of prometheus regexes:
+# git diff origin/drosiek-lint-values deploy/helm/sumologic/values.yaml | \
```

Comment:
I was thinking about automatic formatter for yaml, but yq replaces `|-` with `""`
I could fix this comment to be not depend of this branch